roles/platform/templates/docker-compose.yml.j2

```diff
@@ -15,9 +15,9 @@ services:
     image: {{ ipsec.image }}
     # network_mode: host
     environment:
-      VPN_IPSEC_PSK: 'smartydns'
-      VPN_USER_CREDENTIAL_LIST: '[{"login":"smartUser1","password":"smartpass1"},{"login":"smartUser2","password":"smartPass2"}]'
-      VPN_PUBLIC_IP: 167.71.62.47
+      VPN_IPSEC_PSK: "{{ ipsec.psk }}"
+      VPN_USER_CREDENTIAL_LIST: "{{ ipsec.userlist }}"
+      VPN_PUBLIC_IP: {{ ipsec.public_ip }}
```

Comment:
Please add brackets to the all variables and use the same format as "softethervpn" service has.